packages/react-keytips/src/components/Keytip/renderKeytip.tsx

```diff
@@ -0,0 +1,16 @@
+/** @jsxRuntime classic */
```

Comment:
1. Change pragmas to:

```
/** @jsxRuntime automatic */
/** @jsxImportSource @fluentui/react-jsx-runtime */
```

2. Remove `import { createElement } from '@fluentui/react-jsx-runtime';`

More details https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--docs#rendering-components-with-slots

Reply:
we will have rule for this soon TM https://github.com/microsoft/fluentui/pull/32842

Reply:
discussed offline, context: [this](https://github.com/microsoft/fluentui-contrib/pull/217#discussion_r1755018804) and current storybook webpack & babel aren't working with automatic pragma, needs investigation.

Reply:
can you please open issue for this ? we don't wanna use classic runtime going forward